### DISCUSSION TITLE
建议增加新参数

### DISCUSSION THREAD

**Discussion by fengguowudi** [bohanwood/debi#86] [Ideas]
--china 改为 --ali

--ali

- `--dns '223.5.5.5 223.6.6.6'`
- `--mirror-protocol https`
- `--mirror-host mirrors.aliyun.com`
- `--security-repository mirror`
- `--ntp ntp.aliyun.com`
- `--timezone Asia/Shanghai`

增加：

--tx

- `--dns '119.29.29.29'`
- `--mirror-protocol http`
- `--mirror-host mirrors.tencent.com`
- `--security-repository mirror`
- `--ntp time1.cloud.tencent.com`
- `--timezone Asia/Shanghai`

--tuna

- `--dns '101.6.6.6 166.111.8.28 166.111.8.29'`
- `--mirror-protocol https`
- `--mirror-host mirrors.tuna.tsinghua.edu.cn`
- `--security-repository mirror`
- `--ntp ntp.tuna.tsinghua.edu.cn`
- `--timezone Asia/Shanghai`

--hw

- `--dns '114.114.114.114'`
- `--mirror-protocol https`
- `--mirror-host repo.huaweicloud.com`
- `--security-repository mirror`
- `--timezone Asia/Shanghai`

**Comment by fengguowudi** [comment]
因为有些服务商自己的服务下载会更快

**Comment by bohanwood** [comment]
加上确实会方便一些人，但又不是很想在脚本里堆太多镜像地址……
因为那就相当于管理一个镜像列表了

**Comment by bohanwood** [comment]
添加了中科大和清华的
